src/evt.js

```diff
@@ -36,11 +28,29 @@ evt.prototype.raise = function() {
 };
 
 function evtInit(element) {
-
-  // handle the element/selector
-
-  return new evt();
+  if (!element || (element instanceof Element) === false) {
+    throw new Error('You must supply a DOM element or selector to evt');
+  }
+
```

Comment:
So isn't that how I've done it? Except there needs a check for string.
Then you cant pass any thing other than a string or a instance of Element. Don't think node list is an element.